tests/test_zonemap.py

```diff
@@ -45,74 +46,123 @@ def test_nonexistingzones():
 
 
 def test_errors(tmpdir, caplog):
-    zonefile = tmpdir / "zonemap.lyr"
-    zonefile.write_text(
+    """Test in lyr parse function return correct errors"""
+    lyrfile = tmpdir / "formations.lyr"
+    lyrfile.write_text(
         """
 foo
 """,
         encoding="utf-8",
     )
-    assert ecl2df.common.parse_zonemapfile(zonefile) is None
-    assert "Could not parse zonemapfile" in caplog.text
+    assert ecl2df.common.parse_lyrfile(lyrfile) is None
+    assert "Could not parse lyr file" in caplog.text
     assert "Failed on content: foo" in caplog.text
 
-    zonefile = tmpdir / "zonemap.lyr"
-    zonefile.write_text(
+    lyrfile = tmpdir / "formations.lyr"
+    lyrfile.write_text(
         """
 valid 1-2
 foo 1 2 3
 """,
         encoding="utf-8",
     )
-    assert ecl2df.common.parse_zonemapfile(zonefile) is None
+    assert ecl2df.common.parse_lyrfile(lyrfile) is None
     assert "Failed on content: foo 1 2 3" in caplog.text
 
-    zonefile = tmpdir / "zonemap.lyr"
-    zonefile.write_text(
+    lyrfile = tmpdir / "formations.lyr"
+    lyrfile.write_text(
         """
-valid 1-2 stray
+foo 2-1
 """,
         encoding="utf-8",
     )
-    assert ecl2df.EclFiles(DATAFILE).get_zonemap(str(zonefile)) is None
-    assert "Failed on content: valid 1-2 stray" in caplog.text
+    assert ecl2df.EclFiles(DATAFILE).get_zonemap(str(lyrfile)) is None
+    assert "From_layer higher than to_layer" in caplog.text
+
+    lyrfile = tmpdir / "formations.lyr"
+    lyrfile.write_text(
+        """
+valid 1-2 #FFE5F7
+foo   3- 4 #FFGGHH
+""",
+        encoding="utf-8",
+    )
+    assert ecl2df.EclFiles(DATAFILE).get_zonemap(str(lyrfile)) is None
+    assert "Failed on content: foo   3- 4 #FFGGHH" in caplog.text
```

Comment:
Does this cover the case for a misspelled color, like 'bluez'?

Reply:
I guess not. I will add it